app/dashboard/templates/bounty/details2.html

```diff
@@ -438,7 +438,7 @@ <h5 class="font-body font-weight-semibold">{% trans "SUBMISSIONS" %}</h5>
                             </div>
 
                             <!-- WEB3 MODAL FLOW -->
-                            <div v-else-if="fulfillment.payout_type == 'algorand_ext' || fulfillment.payout_type == 'xinfin_ext' || fulfillment.payout_type == 'rsk_ext' || fulfillment.payout_type == 'web3_modal' || fulfillment.payout_type == 'polkadot_ext' || fulfillment.payout_type == 'harmony_ext' || fulfillment.payout_type == 'binance_ext' || fulfillment.payout_type == 'tezos_ext'" class="px-sm-4">
+                            <div v-else-if="fulfillment.payout_type == 'algorand_ext' || fulfillment.payout_type == 'xinfin_ext' || fulfillment.payout_type == 'rsk_ext' || fulfillment.payout_type == 'web3_modal' || fulfillment.payout_type == 'polkadot_ext' || fulfillment.payout_type == 'harmony_ext' || fulfillment.payout_type == 'binance_ext' || fulfillment.payout_type == 'tezos_ext' || fulfillment.payout_type == 'casper_ext'" class="px-sm-4">
```

Comment:
we are getting in a moment to rethink the crosschain code 🤣

Reply:
swears man! we really need to! 😄